docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,98 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+
+**To customize list icons, add the configuration `{icon="icon-name"}` either after a specific list item or at the end of the entire list.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="fas-fa-times"}
+* item 2
+  * item 2.1 {icon="fas-fa-check"}
+  * item 2.2
+* item 4 {icon="fas-fa-trophy"}
+* item 5 {icon="glyphicon-education"}
+* item 6 {icon="mif-perm-media"}
+* item 7 {icon="zzz"}
+* item 8 {icon="glyphicon-education"}
+* item 9 {icon="octicon-git-pull-request"}
+  </variable>
+</include>
+
```

Comment:
I think at this point we can mention the rules applicable to the above example. We can't expect the reader to read until the end of this feature to find the rules. e.g.,

* Remember that Markdown (and hence, MarkBind) requires an empty line to precede any list.
* If an item consists of multiple lines, the icon specification should be at the end of the first line (see item x in the example above).
* ... other rules ...